Request Validation

aaabramov opened the pull request:
- \+ hibernate validator dependency
- \+ FieldError Pojo
- \+ InvalidRequestProcessing (Advice, Dto, Processor)
- \+ validation_messages file
- \+ WarehouseDto Validation
- \+ some refactoring

aaabramov committed:
+ hibernate validator dependency
+ Request Validation
+ FieldError Pojo
+ InvalidRequestProcessing (Advice, Dto, Processor)
+ validation_messages file
+ WarehouseDto Validation
+ some refactoring

skyfenko commented on web/src/main/java/edu/softserve/zoo/validation/InvalidRequestProcessor.java:
```diff
@@ -0,0 +1,16 @@
+ * @author Andrii Abramov on 6/9/16.
+ */
+public interface InvalidRequestProcessor<T> {
+
```
Write test for this class. Other classes look fine

aaabramov committed:
+ hibernate validator dependency
+ Request Validation
+ FieldError Pojo
+ InvalidRequestProcessing (Advice, Dto, Processor)
+ validation_messages file
+ WarehouseDto Validation
+ some refactoring
+ warehouseDto validation messages

aaabramov committed:
+ hibernate validator dependency
+ Request Validation
+ FieldError Pojo
+ InvalidRequestProcessing (Advice, Dto, Processor)
+ validation_messages file
+ WarehouseDto Validation
+ some refactoring
+ testCompile javax.ex-api
